routes/users.js

```diff
@@ -0,0 +1,78 @@
+const express = require('express');
+const router = express.Router();
```

Comment:
Please keep only imports at the top of the file. Also consider using `import package from 'package'`.